components/roadmap-grid/today-marker.module.css

```diff
@@ -1,7 +1,7 @@
 .todayMarkerWrapper {
   position: absolute;
-  top: 24px;
-  height: calc(100% - 24px);
+  top: 0px;
+  height: calc(100% + 20px);
```

Comment:
What issue is this addressing?

Reply:
some of the updates caused today marker to be displayed incorrectly vertically, addressed in this update as it deals with overall styling but can create new issue to be tracked if needed